### ISSUE TITLE
project:parents is disallowed

### ISSUE THREAD

**Issue by dniku** [GothenburgBitFactory/taskwarrior#2129]
```
$ task add project:parents "Fix grandfather's laptop"
The expression could not be evaluated.
```

_(sigh)_

Occasionally there are some tasks we have to solve that are related to our parents, but creating a project called `parents` is seemingly not allowed. Can this restriction be lifted?

**Comment by pbeckingham** [comment]
This is because `parent` is the name of an attribute, and the argument `project:parents` looks like you are trying to assign to `project` the value of `parent`.  This is obviously not what you are trying to do, but the CLI parsing thinks differently.

This will be addressed in an upcoming release where all DOM access is disambiguated by the `dom.` prefix.

**Comment by andremoeller** [comment]
Just adding a note for other users: the same occurred when I tried to add to a project called "ideas", or any other project "id.+" (I'm guessing from this issue: due to "id" being the name of an attribute):

`task add "blah" project:ideas`
`The expression could not be evaluated.`
`task add "blah" project:ida`
`The expression could not be evaluated.`

**Comment by tbabej** [comment]
Fixed in 2.5.2, 2.6.0.
